web/public/locales/sv.json

```
@@ -950,15 +950,15 @@
     },
     "IT-CNO": {
       "countryName": "Italien",
-      "zoneName": "Centrala norra"
+      "zoneName": "Centrala norra Italien"
     },
     "IT-CSO": {
       "countryName": "Italien",
-      "zoneName": "Centrala södra"
+      "zoneName": "Centrala södra Italien"
```

```suggestion
      "zoneName": "Centrala Italien syd"
```
or
```suggestion
      "zoneName": "Södra centrala Italien"
```

The last one might be a better as it's a little less formal but it's also a tiny bit longer.